drupal/composer.json

```diff
@@ -436,7 +436,10 @@
             "DrupalProject\\composer\\ScriptHandler::removeInternalFiles"
         ],
         "post-autoload-dump": [
-          "cp -rf vendor/simplesamlphp/simplesamlphp web/"
+          "cp -rf vendor/simplesamlphp/simplesamlphp web/",
```

Comment:
This should be replaced with symlink to `vendor/simplesamlphp/simplesamlphp/www`.